deploy/docs/Working_with_container_registries.md

```diff
@@ -46,6 +46,7 @@ Full list of `values.yaml` keys for all the images that are used, can be found b
 | setup job             | `sumologic.setup.job.pullSecrets`               |
 | fluentd               | `sumologic.pullSecrets`                         |
 | Sumo Logic OT distro  | `sumologic.pullSecrets`                         |
+| remote-write-proxy    | `sumologic.pullSecrets`                         |
```

Comment:
please update `README.md` as well

Reply:
I don't think there is anythiing in the README.md to update.